src/Symfony/Component/HttpKernel/Tests/HttpCache/ResponseCacheStrategyTest.php

```diff
@@ -138,6 +138,28 @@ public function testMasterResponseWithExpirationIsUnchangedWhenThereIsNoEmbedded
         $this->assertTrue($masterResponse->isFresh());
     }
 
+    public function testLastModifiedIsMergedWithEmbeddedResponse(): void
```

Comment:
No `void` in test files please to make Fabbot happy, thanks 🙏

Reply:
Fixed in 71db7e7616b6723745ea4ac58f67072a0b52b773